### PR TITLE
[92X] Geometry-independent MillePede validation

### PR DESCRIPTION
**Pull request by ghellwig**
- make the MillePede-internal validation (MPS validation) properly work for phase-1
- MPS validation works now also without checking out the MP package
- added weight information to the MPS validation
- minor bug fixes for exotic use cases

### CODE REVIEW

**Comment by ghellwig** [comment]
please test

File: Alignment/MillePedeAlignmentAlgorithm/python/mpsvalidate/bigStructure.py [hunk 1]
```diff
@@ -75,7 +75,8 @@ def plot(MillePedeUser, alignables, config):
                     if (abs(line.Par[big.data[i]]) != 999999):
                         # set name of the structure
                         big.histoAxis[i].SetBinLabel(
-                            big.binPosition[i], alignables.get_name_by_objid(line.ObjId))
+                            big.binPosition[i],
+                            str(line.Name) if len(line.Name) <= 13 else str(line.Name)[:12]+".")
```
Comment (franzoni) [inline Alignment/MillePedeAlignmentAlgorithm/python/mpsvalidate/bigStructure.py]: hello @ghellwig 
what's special about length 12, in this context ?

Comment (ghellwig) [inline Alignment/MillePedeAlignmentAlgorithm/python/mpsvalidate/bigStructure.py]: I just tuned it such that it fits the available space for the x-axis bin labels with the given text size

**Comment by franzoni** [comment]
+1
no unexpected changes in the relval tests (tools affected are outside standard sequences)

**Comment by ghellwig** [comment]
thanks Giovanni!

**Comment by davidlange6** [comment]
+1

**Pull request merged**